tf2/src/cache.cpp

```diff
@@ -60,6 +60,8 @@ TimeCache::TimeCache(ros::Duration max_storage_time)
 : max_storage_time_(max_storage_time)
 {}
 
```

Comment:
Yes, I think just enclosing everything in a "namespace foo { ... }" is cleaner. It mirrors the pattern in the header file.